SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -8,17 +8,28 @@
 %define config_source %{SOURCE1}
 %endif
 
+%ifarch aarch64
+%global __provides_exclude_from %{_libdir}/debug/.build-id/
+%define arch arm64
+%define archdir arm64
+%define config_source %{SOURCE2}
+%endif
+
+# Generated by update script
+%global kernelver 5.15.98.mshv1
+
 Summary:        Linux Kernel for Kata UVM
 Name:           kernel-uvm
-Version:        5.15.48.1
-Release:        9%{?dist}
+Version:        %{kernelver}
+Release:        1%{?dist}
 License:        GPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Kernel
 URL:            https://github.com/microsoft/CBL-Mariner-Linux-Kernel
-Source0:        https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner-2/%{version}.tar.gz#/kernel-%{version}.tar.gz
+Source0:       %{_mariner_sources_url}/kernel-mshv-%{kernelver}.tar.gz
```

Comment:
this is really minor, but maybe to add a comment why we reference kernel-**mshv** here? It is a bit misleading for people not aware of the details.

In theory we could have named this tar.gz kernel-dom0 and have both kernel-mshv and kernel-uvm packages reference this source. Anyways, not too important, just a suggestion to amke this clear.